cfpb/consumerfinance.gov
# Enforcement actions report

Moving the Enforcement Actions report from a custom python file to a standard Report view inside of the views.py file. The page can be seen from the wagtail admin page and can export a csv file.


---



## Additions

- New Enforcement Actions Report page in the Wagtail admin site


## Removals

-


## Changes

-


## How to test this PR

1.start a local run of cfgov
2. Log into the wagtail admin page3. 
3. Click on Reports and Enforcement actions4. 
4. Click on Download CSV to get an export file and check its contents match the page

## Screenshots
<img width="1344" alt="Screen Shot 2022-07-07 at 11 11 06 AM" src="https://user-images.githubusercontent.com/3359736/177808421-e83daf3d-27a3-47fd-b675-05738c6dc12e.png">


## Notes and todos

-


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
